.github/workflows/title-validation.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
       pull-requests: read
       statuses: write
     steps:
-      - uses: deepakputhraya/action-pr-title@master
+      - uses: deepakputhraya/action-pr-title@tests/typhon-registration
```

Comment:
You should not change this because it refers to the master branch of this repo deepakputhraya/action-pr-title